web/src/app/modules/views/main/editors/modules/graphical-editor/elements/draggable-element-base.ts

```diff
@@ -51,20 +54,37 @@ export abstract class DraggableElementBase<T extends ISpecmatePositionableModelO
         this._rawY = y;
     }
 
+    private userIsDraggingElsewhere = true;
+
+    public get style() {

```

Comment:
I think it's better to solve this with different css classes. This way, we could also remove the reference to the sanitizer.

Additionally, this somehow leads to an error in IE, because the fill value is not interpreted correctly (see general review comments). Additionally, in IE, I had to set the fill value to rgba(0, 0, 0, 0.01) so that the drag events are emitted to the drag handle. With complete transparency (0 alpha), the click did not go to the drag handle.